src/meltano/cli/install.py

```diff
@@ -18,13 +18,15 @@
     from meltano.core.project import Project
     from meltano.core.tracking import Tracker
 
+ANY = "-"
```

Comment:
> https://github.com/meltano/meltano/pull/8228 introduced the --from-file option for meltano config <plugin> set, which accepts a special character - to refer to stdin

The introduction of the special character `-` comes from Click, with the following argument:

> Command line tools are more fun if they work with files the Unix way, which is to accept `-` as a special file that refers to stdin/stdout.

https://click.palletsprojects.com/en/8.1.x/arguments/#file-arguments

Reply:
I wanted to reuse some kind of special character convention that already existed in the CLI - `-` was the only one I could find, but admittedly it's not great as it doesn't have anything to do with stdin/stdout here. `*` would be ideal as it indicates 'any' (i.e. wildcard), but it would have to be escaped in quotes for this kind of usage in Bash.